### PR TITLE
Create faq feature

### PR DESCRIPTION
**Pull request by Evan12Scott**

### CODE REVIEW

**Review by kierankenn — APPROVED** [review]
Looks good!

**Comment by Evan12Scott** [comment]
[Implement the back end for adding a question/answer (including tags)](https://trello.com/c/XstJ2347/32-implement-the-back-end-for-adding-a-question-answer-including-tags)

**Review by bbost14 — COMMENTED** [review]
Nitpicky, but the spacing is inconsistent between some of the variables also missing docstrings.

File: sd_food_bank_ai_bot/admin_panel/forms.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+from django import forms
+from .models import FAQ, Tag
+
+class FAQForm(forms.ModelForm):
```
Comment (kierankenn) [inline sd_food_bank_ai_bot/admin_panel/forms.py:4]: Add some comments!

File: sd_food_bank_ai_bot/admin_panel/views.py [hunk 2]
```diff
@@ -45,3 +47,27 @@ def delete_faq(request, faq_id):
         faq = get_object_or_404(FAQ, id=faq_id)
         faq.delete()
         return redirect('faq_page')
+    
+def create_faq(request):
+    if request.method == "POST":
+        form = FAQForm(request.POST)
+        if form.is_valid():
+            faq = form.save(commit=False)
+            faq.save()
```
Comment (kierankenn) [inline sd_food_bank_ai_bot/admin_panel/views.py:56]: Add some comments here too

**Pull request merged**